ray-operator/controllers/ray/utils/validation_test.go

```diff
@@ -782,7 +782,19 @@ func TestValidateRayServiceSpec(t *testing.T) {
 	require.Error(t, err, "spec.rayClusterConfig.headGroupSpec.headService.metadata.name should not be set")
 
 	err = ValidateRayServiceSpec(&rayv1.RayService{
-		Spec: rayv1.RayServiceSpec{},
+		Spec: rayv1.RayServiceSpec{
+			RayClusterSpec: rayv1.RayClusterSpec{
+				HeadGroupSpec: rayv1.HeadGroupSpec{
+					Template: corev1.PodTemplateSpec{
+						Spec: corev1.PodSpec{
+							Containers: []corev1.Container{
+								{Name: "ray-head"},
```

Comment:
Add non-empty `RayClusterSpec` on existed tests to indicate that it should not be empty. However, there is no code to enforce non-empty `RayClusterSpec` inside `ValidateRayServiceSpec()`. It has marked required in the CRD yaml.